nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -1140,6 +1151,28 @@ private void ResponseChargeActionPoint(ActionBase.ActionEvaluation<ChargeActionP
             }
         }
 
+        private void ResponseMonsterCollect(ActionBase.ActionEvaluation<MonsterCollect> eval)
+        {
+            if (!(eval.Exception is null))
+            {
+                Debug.LogException(eval.Exception);
+                return;
+            }
+
+            NotificationSystem.Push(
+                MailType.System,
+                L10nManager.Localize("UI_MONSTERCOLLECTION_UPDATED"),
+                NotificationCell.NotificationType.Information);
+
+            UpdateAgentStateAsync(eval).Forget();
+            UpdateCurrentAvatarStateAsync(eval).Forget();
```

Comment:
If you want to suppress cautions like this, I suggest you apply this on every `ResponseXXX` method.

Reply:
Um ok. This time, I will revert, and later I will consider other methods together and apply them.